### PR TITLE
OSD-8475

### PR DESCRIPTION
**Pull request by robotmaxtron**
Adding RecentlyActive='PT3H' functionality to the filtering of active volumes to reduce API calls.

### CODE REVIEW

File: monitor/main.py [hunk 1]
```diff
@@ -63,14 +61,14 @@ def collect(aws):
...
+    volumes = set([])
 
     # List of volumes that we've actually had data back for the API
-    seen_volumes = Set([])
+    seen_volumes = set([])
 
     # get the volume IDs and filter the results to show only metrics that have had data points published in the past three hours
     volumePager = cw.get_paginator('list_metrics')
-    for p in volumePager.paginate(MetricName='BurstBalance', Namespace='AWS/EBS'):
+    for p in volumePager.paginate(MetricName='BurstBalance', Namespace='AWS/EBS',RecentlyActive='PT3H'):
```
Comment (mh21) [inline monitor/main.py]:
```suggestion
    for p in volumePager.paginate(MetricName='BurstBalance', Namespace='AWS/EBS', RecentlyActive='PT3H'):
```

File: monitor/main.py [hunk 2]
```diff
@@ -17,7 +15,7 @@
                  "Percent of burstable IOPS credit available", labelnames=['vol_id'])
 
 # A list (implemented as a Set) of all active volumes.
-ACTIVE_VOLUMES = Set([])
+ACTIVE_VOLUMES = set([])
```
Comment (mh21) [inline monitor/main.py]:
```suggestion
ACTIVE_VOLUMES = set()
```

File: monitor/main.py [hunk 3]
```diff
@@ -164,6 +162,8 @@ def collect(aws):
     else:
         session = boto3.session.Session(
             profile_name=args['aws_profile'], region_name=args['aws_region'])
+            #Loading additional AWS models
```
Comment (mh21) [inline monitor/main.py]: I might be missing pieces of the puzzle, but aren't the new images based on Python 3.9 + new botocore, ie they don't need that code anymore (https://github.com/openshift/managed-prometheus-exporter-base/commit/92dabcd9ca384580b931c1631f807ddfb80c9f8d) ?

File: hack/00-osd-managed-prometheus-exporter-ebs-iops-reporter.selectorsyncset.yaml.tmpl [hunk 4]
```diff
@@ -266,7 +270,9 @@ objects:
               - /secrets/aws/credentials.ini
               - -c
               - /config/env/CLUSTERID
-              image: quay.io/openshift-sre/managed-prometheus-exporter-initcontainer:v0.1.9-2019-03-28-4e558131
+              - -m
+              - /config/env/models/cloudwatch/2010-08-01/service-2.json
```
Comment (boranx) [inline hack/00-osd-managed-prometheus-exporter-ebs-iops-reporter.selectorsyncset.yaml.tmpl]: how is this path(and the json) being generated? Nice work in general @robotmaxtron :)

Comment (robotmaxtron) [reply hack/00-osd-managed-prometheus-exporter-ebs-iops-reporter.selectorsyncset.yaml.tmpl]: The path is being generated and the json included from the init container that runs ahead of this "main" container. See: https://github.com/openshift/managed-prometheus-exporter-initcontainer/pull/19

File: monitor/main.py (hunk 3 above)
Comment (robotmaxtron) [reply monitor/main.py]: You're correct, the updated images don't need the additional models. Thank you!

**Comment by robotmaxtron** [comment]
The updated AWS models aren't needed with the newer image versions and have been removed.

**Review by fahlmant — APPROVED** [review]
/lgtm

**Pull request merged**